core/src/apps/bitcoin/keychain.py

```diff
@@ -83,6 +86,9 @@ def validate_path_against_script_type(
         if coin.coin_name in BITCOIN_NAMES:
             patterns.append(PATTERN_GREENADDRESS_A)
             patterns.append(PATTERN_GREENADDRESS_B)
+            patterns.append(PATTERN_UNCHAINED_HARDENED)
```

Comment:
just to confirm: are you sure you are only using **non-segwit** accounts? i.e., script type `SPENDADDRESS`/`SPENDMULTISIG` ?

Reply:
Yes that's right. The recent changes only impact us for non-segwit based accounts.